compiler/rustc_const_eval/src/interpret/memory.rs

```diff
@@ -935,6 +935,10 @@ impl<'tcx, 'a, Tag: Provenance, Extra> AllocRef<'a, 'tcx, Tag, Extra> {
             .check_bytes(&self.tcx, self.range.subrange(range), allow_uninit, allow_ptr)
             .map_err(|e| e.to_interp_error(self.alloc_id))?)
     }
+
+    pub(crate) fn get_relocations(&self, range: AllocRange) -> &[(Size, Tag)] {
+        self.alloc.get_relocations(&self.tcx, self.range.subrange(range))
```

Comment:
You can remove the `range` argument and just always use `self.range` internally. It's the only way this is used anyway.

Reply:
I've amended the commit to do this change.

Thanks oli. Yosh *[is right](https://twitter.com/yoshuawuyts/status/1531654726512693250)*.